src/chapters/aspdotnet-controllers-and-routes/exercises.rst

```diff
@@ -0,0 +1,42 @@
+Exercises: Controllers and Routing
+==================================
+
+.. TODO: check on starter repo name from book content. holding spot name for now: HelloASPDotNET
+
+While reading the chapter, you created a basic Hello, World application using ASP.NET
+called ``HelloASPDotNET``. Open that project up in Visual Studio, and get ready to add some 
+features.
+
+Modify your ``HelloController`` class to display a form on a ``GET``
+request that asks the user for both their name and the language they
+would like to be greeted in. It should look something like this:
```

Comment:
Since this is so similar to what they do in the chapter, could we push them to use attribute routing this time around?

Reply:
now that i've read your lesson content pr - this is assuming the use of attribute routing